### PR TITLE
chore(all packages): fill npmignore to reduce package size

### PR DESCRIPTION
**Pull request by sdwvit**
**What is the problem this PR is trying to solve?**
npm packages have many not-so-useful files that dramatically increase package size when published.

**What is the chosen solution to this problem?**
fill up npmignore configs to reduce package size published on npm. Potentially decrease ci build time. 

**Please check if the PR fulfills these requirements**
- [ ] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [x] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

### CODE REVIEW

File: packages/cmf/.npmignore [hunk 1]
```diff
@@ -37,3 +37,24 @@ jspm_packages
 .node_repl_history
 
 __tests__
+
+# let's reduce npm package size
+storybook-static
+.yarn-tarball.tgz
```
Comment (talend-glorieux) [inline packages/cmf/.npmignore]: Please ignore all `tgz` files.

File: packages/theme/.npmignore [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+node_modules
+jspm_packages
+
+# Optional npm cache directory
+.npm
+
+# Optional REPL history
+.node_repl_history
+
+# let's reduce npm package size
```
Comment (talend-glorieux) [inline packages/theme/.npmignore]: Please change comment.

File: packages/cmf/.npmignore [hunk 3]
```diff
@@ -37,3 +37,23 @@ jspm_packages
...
 __tests__
+
+storybook-static
+*.tgz
+src
+screenshots
+yarn.lock
+stories
+stories-core
+e2e
```
Comment (frassinier) [inline packages/cmf/.npmignore]: AFAIK there is no e2e folder here?

File: packages/cmf/.npmignore [hunk 4]
```diff
@@ -37,3 +37,23 @@ jspm_packages
...
+.sass-lint.yml
+.storybook
+output
+.eslintrc
+examples
+.yarn-metadata.json
+__mocks__
+.editorconfig
+.babelrc
+test
```
Comment (frassinier) [inline packages/cmf/.npmignore]: this file is not really representative of root file hierarchy :/
Should it useful for forms packages?

**Pull request merged**